icepyx/core/query.py

```diff
@@ -917,10 +917,12 @@ def visualize_spatial_extent(
         [visual map output]
         """
 
-        world = gpd.read_file(gpd.datasets.get_path("naturalearth_lowres"))
-        f, ax = plt.subplots(1, figsize=(12, 6))
-        world.plot(ax=ax, facecolor="lightgray", edgecolor="gray")
-        geospatial.geodataframe(self.extent_type, self._spat_extent).plot(
-            ax=ax, color="#FF8C00", alpha=0.7
-        )
-        plt.show()
```

Comment:
Wondering if we should keep this old code as a fallback in case people don't have [`geoviews`](https://geoviews.org) installed. Apparently `pip install geoviews` doesn't work out of the box (see https://geoviews.org/index.html#additional-dependencies) because it needs `PROJ` (which can only be `conda` installed).

Maybe have something like a try...except block or add a parameter (`interactive=True` or `interactive=False`) so that people can choose whether to use an interactive `geoviews` or static `geopandas.plot`? What do you think?

Reply:
Thanks for the advice! Yeah it's a good idea to keep `geopandas.plot` as a backup in case some users can't use geoviews. I think a `try...except` block makes more sense than asking users to add an additional parameter, so if they can use geoviews then it will generate an interactive map, otherwise it will just show a static map.